File: pirates-ios/PiratesGame/Components/Utility/NavigationBarComponent.swift
//
//  NavigationBarComponent.swift
//  Component utility: navigation bar components should subclass this
//  Note: a bar having light content means that a text will be more readable with a white color than black
//

import UIKit

protocol NavigationBarComponent: class {
    
    var isLightContent: Bool { get }
    var isTranslucent: Bool { get }
    
}

File: pirates-android/app/src/main/java/com/crescentflare/piratesgame/components/utility/NavigationBarComponent.kt
package com.crescentflare.piratesgame.components.utility

/**
 * Component utility: action or navigation bar components should implement this
 * Note: a bar having light content means that a text will be more readable with a white color than black
 */
interface NavigationBarComponent {

    val lightContent: Boolean
    val translucent: Boolean

}
